code/messaging/build.gradle.kts

```diff
@@ -86,13 +83,8 @@ dependencies {
     // androidTestImplementation dependencies provided by aep-library:
     // ANDROIDX_TEST_EXT_JUNIT, ESPRESSO_CORE, COMPOSE_UI_TEST_JUNIT4, COMPOSE_UI_TEST_MANIFEST
     androidTestImplementation("com.fasterxml.jackson.core:jackson-databind:2.12.7.1")
-    androidTestImplementation("com.adobe.marketing.mobile:edge:$mavenEdgeVersion") {
-        exclude(group = "com.adobe.marketing.mobile", module = "core")
-        exclude(group = "com.adobe.marketing.mobile", module = "edgeidentity")
-    }
-    androidTestImplementation("com.adobe.marketing.mobile:edgeidentity:$mavenEdgeIdentityVersion") {
-        exclude(group = "com.adobe.marketing.mobile", module = "core")
-    }
```

Comment:
This part wasn't changed in the PR for max width. Is it ok to revert it back?  I think this was done mostly to be able to test with unreleased Core SDK versions